packages/longhorn/longhorn.patch

```diff
@@ -0,0 +1,774 @@
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.nodeDriverRegistrarImage
++  - variable: image.csi.nodeDriverRegistrar.repository
+     default: longhornio/csi-node-driver-registrar
+-    description: "Specify CSI Node Driver Registrar image. Leave blank to autodetect."
++    description: "Specify CSI Node Driver Registrar image repository. Leave blank to autodetect."
+     type: string
+-    label: Longhorn CSI Node Driver Registrar Image
++    label: Longhorn CSI Node Driver Registrar Image Repository
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.nodeDriverRegistrarImageTag
++  - variable: image.csi.nodeDriverRegistrar.tag
+     default: v1.2.0
+     description: "Specify CSI Node Driver Registrar image tag. Leave blank to autodetect."
+     type: string
+     label: Longhorn CSI Node Driver Registrar Image Tag
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.resizerImage
++  - variable: image.csi.resizer.repository
+     default: longhornio/csi-resizer
+-    description: "Specify CSI Driver Resizer image. Leave blank to autodetect."
++    description: "Specify CSI Driver Resizer image repository. Leave blank to autodetect."
+     type: string
+-    label: Longhorn CSI Driver Resizer Image
++    label: Longhorn CSI Driver Resizer Image Repository
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.resizerImageTag
++  - variable: image.csi.resizer.tag
+     default: v0.3.0
+     description: "Specify CSI Driver Resizer image tag. Leave blank to autodetect."
+     type: string
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/templates/_helpers.tpl packages/longhorn/charts/templates/_helpers.tpl
+--- packages/longhorn/charts-original/templates/_helpers.tpl
++++ packages/longhorn/charts/templates/_helpers.tpl
+@@ -36,3 +36,12 @@
+ app.kubernetes.io/instance: {{ .Release.Name }}
+ app.kubernetes.io/version: {{ .Chart.AppVersion }}
+ {{- end -}}
++
++
++{{- define "system_default_registry" -}}
++{{- if .Values.global.systemDefaultRegistry -}}
++{{- printf "%s/" .Values.global.systemDefaultRegistry -}}
++{{- else -}}
++{{- "" -}}
++{{- end -}}
++{{- end -}}
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/templates/clusterrole.yaml packages/longhorn/charts/templates/clusterrole.yaml
+--- packages/longhorn/charts-original/templates/clusterrole.yaml
++++ packages/longhorn/charts/templates/clusterrole.yaml
+@@ -30,11 +30,8 @@
+   verbs: ["*"]
+ - apiGroups: ["coordination.k8s.io"]
+   resources: ["leases"]
+-  verbs: ["get", "watch", "list", "delete", "update", "create"]
++  verbs: ["*"]
+ - apiGroups: ["longhorn.io"]
+   resources: ["volumes", "volumes/status", "engines", "engines/status", "replicas", "replicas/status", "settings",
+               "engineimages", "engineimages/status", "nodes", "nodes/status", "instancemanagers", "instancemanagers/status"]
+   verbs: ["*"]
+-- apiGroups: ["coordination.k8s.io"]
+-  resources: ["leases"]
+-  verbs: ["*"]
```

Comment:
this either, I'm assuming the operator still needs this in clusterrole to function